lib/tasks/puppet.rake

```diff
@@ -57,4 +57,54 @@ namespace :puppet do
       end
     end
   end
+
+  namespace :migrate do
+    desc "
+    Migrate Puppet provisioning parameters from ENC to Host parameters.
+    This task is relevant if you want to continue using Puppet provisioning,
+    but do not want to install the Puppet plugin and get the full ENC funcionality."
+
+    task :to_parameters => :environment do
+      class FakeEnv < ::ApplicationRecord
+        self.table_name = 'environments'
+      end
+
+      def add_host_param(host, name, value)
+        param = HostParameter.find_by(host: host, name: name)
+        if !param
+          HostParameter.create(host: host, name: name, value: value)
+        elsif param.value != value
+          puts "Parameter '#{name} = #{value}' for Host '#{host.name}' could not be defined because it is already defined on Host with value #{param.value}"
+        end
+      end
+
+      def add_hostgroup_param(hostgroup, name, value)
+        param = GroupParameter.find_by(host: hostgroup, name: name)
+        if !param
+          GroupParameter.create(host: hostgroup, name: name, value: value)
+        elsif param.value != value
+          puts "Parameter '#{name} = #{value}' for Hostgroup '#{hostgroup.name}' could not be defined because it is already defined on Hostgroup with value #{param.value}"
```

Comment:
```suggestion
          puts "Parameter '#{name} = #{value}' for Hostgroup '#{hostgroup.title}' could not be defined because it is already defined on Hostgroup with value #{param.value}"
```